src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -200,7 +200,7 @@ public function testValidateCustomErrorPath()
         $this->buildViolation('myMessage')
             ->atPath('property.path.bar')
             ->setParameter('{{ value }}', '"Foo"')
-            ->setInvalidValue('Foo')
+            ->setInvalidValue($entity2)
```

Comment:
Ah true. Sorry got confused because the entities have a `__toString()` method that returns the name :blush: So indeed this seems to be the cleaner solution.